File: projetMusicBoot/src/main/java/formation/sopra/projetMusicBoot/entities/Genre.java
package formation.sopra.projetMusicBoot.entities;

public enum Genre {
	Rock("Rock"), HunnuRock("Hunnu Rock"), Metal("Metal"), TechnicalDeathcore("Technical Deathcore"), MelodicDeath("Melodic Death"), ProgressiveMelodicDeath("Progressive Melodic Death"), Classical("Classical"), Electro("Electro"), Pop("Pop"), Rap("Rap"), RnB("RnB"), VarieteFrancaise("VarieteFrancaise");
	
	private String name;

	private Genre(String name) {
		this.name = name;
	}

	private Genre() {
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
	
	
}

File: projetMusicAngular/src/app/model/genre.ts
export enum Genre {
  ROCK = 'Rock',
  HUNNUROCK = 'Hunnu Rock',
  METAL = 'Metal',
  TECHNICALDEATHCORE = 'Technical Deathcore',
  MELODICDEATH = 'Melodic Death',
  PROGRESSIVEMELODICDEATH = 'Progressive Melodic Death',
  CLASSICAL = 'Classical',
  ELECTRO = 'Electro',
  POP = 'Pop',
  RAP = 'Rap',
  RNB = 'RnB',
  VARIETEFRANCAISE = 'Variete Francaise',
}

export const GenreLabelMapping: Record<Genre, string> = {
  [Genre.ROCK]: 'Rock',
  [Genre.HUNNUROCK]: 'Hunnu Rock',
  [Genre.METAL]: 'Metal',
  [Genre.TECHNICALDEATHCORE]: 'Technical Deathcore',
  [Genre.MELODICDEATH]: 'Melodic Death',
  [Genre.PROGRESSIVEMELODICDEATH]: 'Progressive Melodic Death',
  [Genre.CLASSICAL]: 'Classical',
  [Genre.ELECTRO]: 'Electro',
  [Genre.POP]: 'Pop',
  [Genre.RAP]: 'Rap',
  [Genre.RNB]: 'RnB',
  [Genre.VARIETEFRANCAISE]: 'Variete Francaise',
};
